docs/ldml/tr35-keyboards.md

```diff
@@ -981,6 +987,8 @@ _Attribute:_ `base`
 _Attribute:_ `path` (required)
 
 > If `base` is `cldr`, then the `path` must start with a CLDR version (such as `techpreview`) representing the CLDR version to pull imports from. The imports are located in the `keyboard/import` subdirectory of the CLDR source repository.
+> Implementations are not required to have all CLDR versions available to them, and an implementation may reject a keyboard file which references an unavailable CLDR version.
```

Comment:
Would not support a keyboard that

Reply:
@macchiati is that to replace "may reject"? then that is a stronger statement, yet more of a claim rather than instruction. Currently it reads as "implementation can inform the user about unavailability of required data and let him decide to go ahead with version that is available". If your intention is taht implementation shall not allow this, than it needs to be stated more explicity.

I would also change "all CLDR versions" to "all or any CDLR versions".

Reply:
I think it's still may, because it should be up to the implementation.

Reply:
i've already dropped the second clause.